core/domain/stats_jobs_one_off.py

```diff
@@ -1321,31 +1287,19 @@ def map(exp):
 
                 # Is a revert commit.
                 revert_to_version = commit_log.commit_cmds[0]['version_number']
-                try:
-                    stats_services.handle_stats_creation_for_new_exp_version(
-                        exp.id, version, exp_at_version.states, None,
-                        revert_to_version)
-                except Exception:
```

Comment:
`handle_stats_creation_for_new_exp_version` can raise an exception under an invalid id, version, revert_to_version or states, but these values are derived above and hence they cannot be invalid. This is why I removed this exception